crates/oxc_linter/src/snapshots/no_aria_hidden_on_focusable.snap

```diff
@@ -0,0 +1,47 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_aria_hidden_on_focusable
+---
+  ⚠ eslint-plugin-jsx-a11y(no-aria-hidden-on-focusable): `aria-hidden` must not be true on focusable elements.
+   ╭─[no_aria_hidden_on_focusable.tsx:1:1]
+ 1 │ <div aria-hidden="true" tabIndex="0" />;
+   · ───────────────────────────────────────
+   ╰────
+  help: Remove `aria-hidden="true"` from focusable elements or modify the element to be not focusable.
```

Comment:
do you mind making this more specific and point to `aria-hidden="true"` instead?

Reply:
@camc314 
Thank you for your review! I've updated the implementation to ensure the error specifically points to aria-hidden="true" https://github.com/oxc-project/oxc/pull/1795/commits/ebeb3b9101fcd083f888362eeeb7d388117a0483. However, I noticed that the original implementation in eslint-plugin-jsx-a11y uniformly uses the message `aria-hidden="true" must not be set on focusable elements.` [Reference](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/__tests__/src/rules/no-aria-hidden-on-focusable-test.js#L21C1-L24C3). 
Could you please specify how you would like this to be more specific? 🙏

Reply:
yep that's perfect, thank you